docs/src/hooks/use-on-document-click.ts

```diff
@@ -0,0 +1,18 @@
+import { useEffect } from 'react'
+import { canUseEventListeners } from 'exenv'
+
+export default function useOnDocumentClick(
```

Comment:
I think you could use this instead: https://github.com/pluralsight/design-system/blob/master/packages/util/src/use-close-on-document-events.js

Reply:
yep. these are all copied from other parts of the project. i'll follow up with a pr that centralizes them